source/Symbol/SwiftASTContext.cpp

```diff
@@ -1705,12 +1713,12 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(lldb::LanguageType language,
     bool got_serialized_options;
     llvm::SmallString<0> error;
     llvm::raw_svector_ostream errs(error);
-
-    if (DeserializeAllCompilerFlags(*swift_ast_sp, module, m_description, errs,
+    Process *process = target ? target->GetProcessSP().get() : nullptr;
+    if (DeserializeAllCompilerFlags(*swift_ast_sp, process, module,
```

Comment:
I doesn't look to me like DeserializeAllCompilerFlags actually uses the process, so maybe you don't need to do this?